test/test_nestedtensor.py

```diff
@@ -300,6 +300,72 @@ def test_nested_namespace(self):
         nested_namespace_result = torch.nested.to_padded_tensor(nt, 4)
         self.assertEqual(result, nested_namespace_result)
 
+    # Helper function to generate a random nested tensor
+    def random_nt(self, device, dtype, num_tensors, max_dims, min_dims=None):
```

Comment:
Isn't there already a helper defined like this in this test file. Maybe it is just a class method and if so feel free to extract out a layer so that you can access it